packages/augur-ui/src/modules/market/components/common/social-media-buttons.tsx

```diff
@@ -32,17 +35,19 @@ export const SocialMediaButtons = (props: SocialMediaButtonsProps) => {
     <>
       <button
         id="facebookButton"
-        onClick={() =>
-          showFacebookShare(encodedMarketUrl, encodedMarketDescription)
-        }
+        onClick={() => {
+          props.sendFacebookShare(props.marketAddress, props.marketDescription);
+          showFacebookShare(encodedMarketUrl, encodedMarketDescription);
```

Comment:
this looks weird can ` showFacebookShare(encodedMarketUrl, encodedMarketDescription);` be in the props.sendFacebookShare ... method